manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Automatically updated by the winget bot at 2021/Jul/28
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+PackageIdentifier: Microsoft.SQLServerManagementStudio
+PackageVersion: 15.0.18386.0
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://docs.microsoft.com/sql/ssms/download-sql-server-management-studio-ssms
+Author: Microsoft
+PackageName: Microsoft SQL Server Management Studio
+PackageUrl: https://docs.microsoft.com/sql/ssms/download-sql-server-management-studio-ssms
+License: Copyright (c) Microsoft Corporation
+Copyright: Copyright (c) Microsoft Corporation
+ShortDescription: Microsoft.SQLServerManagementStudio
```

Better ShortDescription
```suggestion
ShortDescription: SQL Server Management Studio (SSMS) is an integrated environment for managing any SQL infrastructure
```